src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int i = 0;
+        Ball ball = new Ball();
+        while (i < 3) {
```

Comment:
`3` should be a constant field because it is a magic number

also, use `for` loop instead